sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,4 @@
+select "storeId",
+        count("filmId") as "filmCt"
+  from "inventory"
+group by "storeId"
```

Comment:
While it looks like you got the hang of `count`, the prompt was asking for a single total of all the film copies, not grouped per-store.

---

![image](https://user-images.githubusercontent.com/16143771/131198569-d8c3da55-31c9-4aca-a2bd-e02e9c8f79bf.png)